File: C/Ex27.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
  
  int numero, r, n1, n2;
 
  printf("Digite um numero positivo: ");
  scanf("%i", &numero);
 
  while(numero <= 0){
    printf("Erro! Apenas numeros positivos. Digite novamente: ");
    scanf("%i", &numero);
}
  if  (numero > 0) {printf("Digite o primeiro numero: ");
  scanf("%i", &n1);
  printf("Digite o segundo numero: ");
  scanf("%i" ,&n2); 
  while(n1 >= n2){
    printf("Digite novamente o segundo numero: ");
    scanf("%i", &n2);
  }
}

for (int i=n2; i>=n1; i--){
    r = numero * i;
    printf("%i x %i = %i\n", numero, i, r);
}
return 0;

  
}

File: Python/Ex27.py
n=int(input("Digite um numero positivo: "))
while(n<=0):
    print("Erro! Apenas numeros positivos.")
    n=int(input("Digite novamente: "))
if (n>0):
    n1=int(input("Digite o primeiro numero: "))
    n2=int(input("Digite o segundo numero: "))
while(n1 >= n2):
    n2=int(input("Digite novamente o segundo numero: "))
for i in range (n2, n1-1, -1):
    r=n*i
    print(n,"*", i, "=", r)

